thrust/system/detail/generic/shuffle.inl

```diff
@@ -99,10 +99,11 @@ class feistel_bijection {
   // from random for each key value input. This is not cryptographically secure
   // but sufficient for generating permutations. 
   __host__ __device__ uint32_t round_function(uint64_t value,
-                                              const uint64_t key) const {
-    uint64_t hash0 = thrust::random::taus88(value)();
+                                              const uint64_t key_) const {
+    uint64_t hash0 = thrust::random::taus88(static_cast<uint32_t>(value))();
     uint64_t hash1 = thrust::random::ranlux48(value)();
-    return hash_combine(hash_combine(hash0, key), hash1) & left_side_mask;
+    return static_cast<uint32_t>(
```

Comment:
Ditto here, can `uint32_t` and `uint64_t` be `std::uint32_t` and `std::uint64_t`?

Reply:
Done.